commons/error_messages.py

```diff
@@ -52,6 +52,14 @@
                                "operation: This copy request is illegal because it is trying to "
                                "copy an object to itself without changing the object's metadata, "
                                "storage class, website redirect location or encryption attributes.")
+RGW_ERR_CPY_IF_MATCH_FALSE  = ("An error occurred (PreconditionFailed) when calling the "
+                               "CopyObject operation: Unknown")
+RGW_ERR_CPY_IF_NONE_MATCH_FALSE  = ("An error occurred (304) when calling the CopyObject "
+                                     "operation : Not Modified")
```

Comment:
Please fix indentation in code block 58 - 62

Reply:
yes